docker/py-dsci-100/Dockerfile

```diff
@@ -24,7 +24,6 @@ RUN pip install --upgrade git+https://github.com/scikit-learn/scikit-learn@7de59
 RUN mamba install --quiet --yes \
         'numpy' \
         'jinja2' \
-        'altair_data_server' \
```

Comment:
Does `altair` install this already? or should it be added below as well

Reply:
We no longer need the data server. It was used to handle large data sets but vegafusion replaces it. I don't think we really used it previously either since we just went with `disable_max_rows`. This line also installed `altair_saver` as a dependency, but the saving is now handled by `vl-convert-python` so we don't need that either.

Reply:
super, thank you!